src/actinia_core/endpoints.py

```diff
@@ -101,6 +102,9 @@ def create_core_endpoints():
     flask_api.add_resource(
         MapsetLockManagementResource,
         '/locations/<string:location_name>/mapsets/<string:mapset_name>/lock')
+    flask_api.add_resource(
+        MapsetLockManagementResourceAdmin, '/locations/locks')
```

Comment:
Usually in actinia, after `/locations/` the adressed location name is followed. As this makes no sense in this case, I would prefer a route without `locations` in it at all. I cannot think of a good name yet but will try again tommorow :upside_down_face:

Reply:
Just for documentation: We agreed upon a new Resource `/mapsets` independant of a location. For now, it will only be accessible by HTTP GET parameter filter and empty response without filter.

The class will be moved away from `mapset_management.py` into a new one.